docs/community/com-monitoring/old-monitoring.md

```diff
@@ -6,6 +6,8 @@ grand_parent: Community
 nav_order: 20
 ---
 
+NOTE: All but the scaling has been
```

Comment:
has been..... done?

Reply:
Yes, though the file is in the wrong place. And it needs a *lot* more detail to be actually useful.

File is in /com-config/com-config-fb-scaling

It should be under clustering because that's what it's referring to.